packages/ui/src/styles/main.css

```diff
@@ -1,3 +0,0 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
```

Comment:
So the solution was to _remove_ `styles/main.css` from the UI package? I did not try that! Thanks for cleaning up my mess, @VanishMax. :innocent: